include/wil/Microsoft.Windows.Telemetry.Resource.h

```diff
@@ -0,0 +1,170 @@
+#pragma once
+//
+//! @file
+//! WIL Resource Wrappers (RAII): Provides a family of smart pointer patterns and resource wrappers to enable customers to
+//! consistently use RAII in all code.
+
+// Common.h contains logic and configuration macros common to all consumers in Windows and therefore needs to be
+// included before any 'opensource' headers
+#include "Microsoft.Windows.Telemetry.Common.h"
+
+#include <wil/resource.h>
+
+// Internal WIL header files that were historically included before WIL was split into public and internal versions.
+// These exist only to maintain back-compat
+#include "Microsoft.Windows.Telemetry.ResultMacros.h"
+
+#pragma warning(push)
+#pragma warning(disable:26135 26110)    // Missing locking annotation, Caller failing to hold lock
+#pragma warning(disable:4714)           // __forceinline not honored
+
+#ifndef __MICROSOFT_WINDOWS_TELEMETRY_RESOURCE
+#define __MICROSOFT_WINDOWS_TELEMETRY_RESOURCE
+namespace wil
+{
+    /// @cond
+#ifndef WIL_HIDE_DEPRECATED_1611
+    WIL_WARN_DEPRECATED_1611_PRAGMA(DetachToOptParam)
+
+        // DEPRECATED: Use wil::detach_to_opt_param
+        template <typename T, typename TSmartPointer>
+    inline void DetachToOptParam(_Out_opt_ T* outParam, TSmartPointer&& smartPtr)
+    {
+        detach_to_opt_param(outParam, wistd::forward<TSmartPointer>(smartPtr));
+    }
+#endif
+    /// @endcond
+}
+#endif // __MICROSOFT_WINDOWS_TELEMETRY_RESOURCE
+
+namespace wil
+{
+#ifdef __WIL_WINBASE_
+
+#ifndef __WIL_INTERNAL_WINBASE_
+#define __WIL_INTERNAL_WINBASE_
+#ifndef WIL_HIDE_DEPRECATED_1611
+    WIL_WARN_DEPRECATED_1611_PRAGMA(unique_hheap_ptr)
+
+        // deprecated, use unique_process_heap_ptr instead as that has the correct name.
+        template <typename T>
+    using unique_hheap_ptr = wistd::unique_ptr<T, process_heap_deleter>;
+#endif
+#endif // __WIL_INTERNAL_WINBASE_
+#endif // __WIL_WINBASE_
+
+    // do not add new uses of UNIQUE_STRING_VALUE_EXPERIEMENT, see http://osgvsowi/11252630 for details

```

Comment:
Seems everything in this header is also either deprecated or has guidance not to use. This file should also be deleted